css/store2.css

```
@@ -0,0 +1,254 @@
+main {
+    width: 100vw;
+    flex-direction: column;
+}
+
+
+/* Form Section */
+
+.form-side {
+    width: 100%;
+    box-sizing: border-box;
+    padding: 0 30px;
+}
+
+h2 {
+    color: #e0633a;
+    padding-top: 0;
+    font-size: 35px;
+}
+
+
+/* Container for the form rows */
+
+fieldset {
+    flex-direction: column;
+    border: none;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+legend {
+    margin: 10px 0;
+}
+
+
+/* Container for the label and input */
+
+.col {
+    flex-direction: column;
+    width: 100%;
+}
+
+.name {
+    width: 100%;
+    margin-right: 0%;
+}
+
+.name:first-of-type .half-width {
+    width: 100%;
+}
+
+.full-width {
+    width: 100%;
+}
+
+label {
+    width: fit-content;
+}
+
+input {
+    width: 100%;
+    padding: 7px;
+    margin-bottom: 20px;
+    border-radius: 5px;
+    border: 1px solid grey;
+    box-sizing: border-box;
+}
+
+
```

```suggestion
input[type="submit"] {
     width: 30%
}
```

I would add this to make sure the button isn't full width (it shouldn't be according to design)